tox.ini

```diff
@@ -13,7 +13,7 @@ setenv =
     numpydev: PIP_EXTRA_INDEX_URL = https://pypi.anaconda.org/scientific-python-nightly-wheels/simple
 description = run tests
 deps =
-    pytestoldest: pytest==4.6.0
+    pytest46: pytest==4.6.*
```

Comment:
The oldest have to stay explicit and the version that is declared in the config. Besides that, there is no need to test the latest bugfix in the 4.6 series